lib/pid/include/PID.h

```diff
@@ -1,282 +1,61 @@
 /**
- * @author Aaron Berk
+ * This PID library was inspired by http://brettbeauregard.com/blog/2011/04/improving-the-beginners-pid-introduction/
+ * Terminology:
+ *  Plant: What you are trying to control
+ *  Process Variable(PV): Measured value of plant. Sensor reading.
```

Comment:
measurement might be better than referring to it as PV. idk

Reply:
i think theyd both be correct and pv is what ppl would see if they googled pid diagram so i think we should leave it as pv if thats alright